src/DynamoCoreWpf/ViewModels/Preview/WatchViewModel.cs

```diff
@@ -211,6 +218,37 @@ public WatchViewModel(string label, string path, Action<string> tagGeometry, boo
             numberOfItems = 0;
             maxListLevel = 0;
             isCollection = label == WatchViewModel.LIST || label == WatchViewModel.DICTIONARY;
+            ValueType = GetDisplayType(outputType);
+        }
+
+        private string GetDisplayType(TypeCode? typecode)
+        {
+            switch (typecode)
+            {
+                case TypeCode.Boolean:
+                    return "bool";
```

Comment:
toString() does not suffice?

Reply:
could do that, but the labels would be something like:

- `String`
- `Int64`
- `Double`
- `Object`
- `Boolean`

Instead of:
- `string`
- `int`
- `double`
- `object`
- `bool`

Reply:
gotcha, this is fine with me then, can you add a comment noting you wanted customized user friendly strings?